mirrord-layer/src/pod_api.rs

```diff
@@ -110,6 +105,7 @@ pub async fn create_agent(config: LayerConfig) -> Result<Portforwarder, LayerErr
             &impersonated_pod_namespace,
             &impersonated_container_name,
         )
+        .unwrap()
         .await;
```

Comment:
ah, no need to unwrap as you can just propogate, sorry about the bad advice.
try
`.await?;` instead

Reply:
no worries. I am also flying in the dark here.